vcpkg/users/platforms/android.md

```diff
@@ -150,68 +160,29 @@ The package vulkan-hpp:arm64-android is header only and can be used from CMake v
 
 </details>
 
-## Consume libraries using vpckg, cmake and the android toolchain
```

Comment:
Why is this section no longer required?

Reply:
Now, the android toolchain `android.cmake` directly inspects `ANDROID_NDK_HOME` to include the Android NDK toolchain, so there is no need to add it separately. Also, settings such as `ANDROID_ABI` are set in the triplet.